sdk/translation/ai-translation-text-rest/src/index.ts

```diff
@@ -1,16 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import TextTranslation from "./customClient";
+import TextTranslationClient from "./textTranslationClient";
 
-export * from "./customClient";
-export * from "./generated/parameters";
-export * from "./generated/responses";
-export * from "./generated/clientDefinitions";
-export * from "./generated/isUnexpected";
-export * from "./generated/models";
-export * from "./generated/outputModels";
-export * from "./generated/serializeHelper";
-export { TranslatorCredential } from "./authentication";
```

Comment:
We need to keep auth if the file is to be used - see my comment

Reply:
Restored.

Reply:
You are missing `export * from "./customClient";` which will cause compilation error.